plugins/modules/mount.py

```diff
@@ -113,10 +113,13 @@
     mount_dir: /dev/hd1
     mount_over_dir: /home
 
-- name: Mount filesystems provided by a node"
+- name: |
+    Mount a filesytem, /tmp/servnfs provided by
+    a node on the mount point /tmp/clientnfs"
```

Comment:
Is there a particular reason of changing the mount directory from /mnt to /tmp? /mnt is AIX default mount and usually /tmp has small size with default install. what are the consideration of the changes? is this recommendation for user?

Reply:
there is no particular reason. this is the mount point of `filesystem` parameter. size is determined by the backing LV of the `filesystem` and not the size of the rootvg filesystem `/tmp`
however, i can make the mount point back to `/mnt`, i was just used to making the mount point that way during testing with filesystem

Reply:
We might want to keep the same for the shipped product if there is no errors or issues so we can avoid any unnecessary confusions to users

Reply:
i reverted back to `/mnt` for the example text